couchbase/analytics_index_manager.hxx

```diff
@@ -91,7 +91,7 @@ public:
    * @since 1.0.0
    * @committed
    */
-  void drop_dataverse(std::string dataverse_name,
+  void drop_dataverse(const std::string& dataverse_name,
```

Comment:
Is turning `std::string` to `const std::string&` a breaking change?

Reply:
Technically it is, as the compiled code will not be able to find the signature as it is using different type. But I think I'd rather make it early and right. Right now we do not ship binaries anyway yet. And the user would need to recompile it anyway.